ghcide/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -739,7 +739,7 @@ suggestConstraint df parsedModule diag@Diagnostic {..}
     where
       findMissingConstraint :: T.Text -> Maybe T.Text
       findMissingConstraint t =
-        let regex = "(No instance for|Could not deduce) \\((.+)\\) arising from a use of"
+        let regex = "(No instance for|Could not deduce) \\((.+)\\) arising from" -- a use of / a do statement
```

Comment:
why not push this into the regex? feels weird to have it as a comment next to the regex.

Reply:
I did not push it into the regex because I don't know if there are other suffixes to this sentence that could result in more opportunities for a quick fix. 

Why is it weird to have a comment next to a regex? We do it in most regexes in this module

Reply:
It feels weird because you are saying "after this regex comes one of these two suffixes". But what you really mean is these two values have been observed in the wild, there might be more, who knows.